lighthouse-core/report/html/report-styles.css

```diff
@@ -544,50 +544,33 @@
   display: none;
 }
 
-
 /* Perf Metric */
 
-.lh-columns {
-  display: flex;
-  width: 100%;
-}
-@media screen and (max-width: 640px) {
-  .lh-columns {
-    flex-wrap: wrap;
-
-  }
+.lh-metrics-container {
+  display: grid;
+  grid-template-rows: 1fr 1fr 1fr;
```

Comment:
dunno a lot about grid but does this need to be repeated 3 times?  what if another metric is added via custom config or w/e. do we need changes here?

Reply:
right, a 7th metric would make a new column and be weird ...

if instead i used `grid-template-columns: 1fr 1fr` (and drop the `grid-flow: columns` so it defaults to `rows`) the order is no longer we want:

![image](https://user-images.githubusercontent.com/4071474/81750694-de916d80-9462-11ea-85de-f1a685fca4b5.png)

we forced this order before by using  two divs, but that didn't allow for alignment.

Reply:
idk how to solve this other than keeping as-is and being ok with this in for some reason there is another metric in the LHR 

![image](https://user-images.githubusercontent.com/4071474/81750798-0a145800-9463-11ea-9288-f7fbbaa9da4c.png)


I think we'd redesign this whole thing if we had >6 metrics here tho..

Reply:
another solution: interleave the creation of these `lh-metric` divs such that the rendering order is good. like `1 4 2 5 3 6` ...

Reply:
in other words, css grid doesnt allow for letting elements fill up a column 1 first, column 2 second, and then not create a 3rd column (what would that css even look like...)

Reply:
gotcha. IMO with 7 metrics it doesnt look that bad. 

I'm fine with keeping the css as you have it.

Reply:
cool :beans: